tests/nightly/Jenkinsfile

```diff
@@ -120,7 +120,15 @@ core_logic: {
           utils.docker_run('ubuntu_nightly_cpu', 'nightly_test_javascript', false)
         }
       }
-    }
+    },
+    'Clojure Nightly: CPU': {
```

Comment:
This has to be in tests/nighty/JenkinsfileForBinaries since Clojure requires a compiled version of MXNet while this file is intended for source versions of MXNet.